www/tests/test_app.py

```diff
@@ -1,4 +1,11 @@
 import app
 
-def test_dummy():
+def test_sensor_init():
     assert len(app.binary_sensors) > 0
+
+def test_get_local_ip():
```

Comment:
I wrapped it in a class with a stop function that disabled the `shutdown` variable. And socketio has not problem accepting a class member.

But, in my test function, how do I get the result of the thread function? If I start it in the background using socketio, I don't know where the return goes to. If I start it in the foreground, it blocks and I can't call the .stopThread function.